java-checks/src/main/java/org/sonar/java/checks/security/CookieHttpOnlyCheck.java

```diff
@@ -71,20 +71,25 @@ public class CookieHttpOnlyCheck extends IssuableSubscriptionVisitor {
 
   private static final class ClassName {
     private static final String SERVLET_COOKIE = "javax.servlet.http.Cookie";
+    private static final String JAKARTA_SERVLET_COOKIE = "jakarta.servlet.http.Cookie";
     private static final String NET_HTTP_COOKIE = "java.net.HttpCookie";
     private static final String JAX_RS_COOKIE = "javax.ws.rs.core.Cookie";
+    private static final String JAKARTA_RS_COOKIE = "jakarta.ws.rs.core.Cookie";
     private static final String JAX_RS_NEW_COOKIE = "javax.ws.rs.core.NewCookie";
+    private static final String JAKARTA_RS_NEW_COOKIE = "jakarta.ws.rs.core.NewCookie";
     private static final String SHIRO_COOKIE = "org.apache.shiro.web.servlet.SimpleCookie";
     private static final String PLAY_COOKIE = "play.mvc.Http$Cookie";
     private static final String PLAY_COOKIE_BUILDER = "play.mvc.Http$CookieBuilder";
   }
 
   private static final Set<String> SETTER_NAMES = SetUtils.immutableSetOf("setHttpOnly", "withHttpOnly");
 
-  private static final Set<String> CLASSES = SetUtils.immutableSetOf(
```

Comment:
Is there a specific reason to change this? I think it's good to enforce that the set is not changed by the logic of the rule at any point

Reply:
`Set.of` is more efficient and it returns an immutable collection, it is also from `java.util` that's more convenient :)